tests/run.sh

```diff
@@ -152,41 +142,17 @@ function get_requirements() {
     chmod +x downloaded-tools/single-file-artifact-gen
 
     export PATH=$PWD/downloaded-tools:$PATH
-
-    inject_pre_generated_ssh_keys
```

Comment:
Q: Is this supposed to be a part of this commit?

Reply:
No :)

Reply:
Rhetorical question :smile: